pyramid_jinja2/scaffolds/jinja2_starter/+package+/templates/mytemplate.jinja2_tmpl

```diff
@@ -38,15 +38,15 @@
                 <span class="smaller">Jinja2 scaffold</span>
               </h1>
               <p class="lead">
-                {% trans %}Hello{% endtrans %} to <span class="font-normal">\{\{ project \}\}</span>, an&nbsp;application generated&nbsp;by<br>the <span class="font-normal">Pyramid Web Framework 1.5.4</span>.</p>
+                {% trans %}Hello{% endtrans %} to <span class="font-normal">\{\{ project \}\}</span>, an&nbsp;application generated&nbsp;by<br>the <span class="font-normal">Pyramid Web Framework {{pyramid_version}}</span>.</p>
```

Comment:
where is `{{pyramid_version}}` coming from? Does it need to be escaped like `\{\{ project \}\}`? Happy to merge if you tested this looks OK